tutorials/hello/hello_client/src/app/plugins/hello_info/db.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { codec, db as liskDB, cryptography } from 'lisk-sdk';
+import * as os from 'os';
+import { join } from 'path';
+import { ensureDir } from 'fs-extra';
+import { offChainEventSchema, counterSchema, heightSchema } from './schemas';
+import { Event, Counter, Height } from './types';
+import { DB_KEY_ADDRESS_INFO, DB_LAST_COUNTER_INFO, DB_LAST_HEIGHT_INFO } from './constants';
+
+const { Database } = liskDB;
+type KVStore = liskDB.Database;
+
+// Returns DB's instance.
+export const getDBInstance = async (
+    dataPath: string,
+    dbName = 'lisk-framework-helloInfo-plugin.db',
+): Promise<KVStore> => {
+    const dirPath = join(dataPath.replace('~', os.homedir()), 'plugins/data', dbName);
```

Comment:
`dataPath` in the input should have the whole path from the root. Please check the input here

Reply:
I did check and the folders are getting created in the correct place, Although, I have tweaked the path a little to make it look like this:
```
iamtalha@Muhammads-MacBook-Pro ~ % ls .lisk/hello_client/plugins/helloInfo/data/database/lisk-framework-helloInfo-plugin.db 
000005.log	CURRENT		IDENTITY	LOCK		LOG		MANIFEST-000004	OPTIONS-000007
```